### ISSUE TITLE
How to use data_flow.FeedDictFlow?

### ISSUE THREAD

**Issue by lfwin** [tflearn/tflearn#668]
I want to just use class FeedDictFlow, ImagePreprocessing and ImageAugmentation.  but how to let test data set and validating date set to share the same parameters such as mean and variviance of training data.
this is outline of my code:
`
X_train, y_train, X_test, y_test = dataset.cifar_100()
image_batch = tf.placehoder()
label_batch = tf.placehoder()
feed_dict = {image_batch:X_train, label_batch:y_train}
test_feed_dict = {image_batch:X_test, label_batch:y_test}
loss, acc = model(image_batch, label_batch) # define a neural net
img_prep = tflearn.ImagePreprocessing()
img_prep.add_featurewise_zero_center()

img_aug = tflearn.ImageAugmentation()
img_aug.add_random_flip_leftright()

with tf.Session() as sess:
     coord = tf.train.Coordinator()
     df = data_flow.FeedDictFlow(feed_dict, coord,
                                    batch_size=batch_size,
                                    dprep_dict=img_prep,
                                    daug_dict=img_aug,
                                    index_array=None,
                                    num_threads=1)
    df.start()
    
    test_df = data_flow.FeedDictFlow(test_feed_dict, coord,
                                    batch_size=batch_size,
                                    dprep_dict=img_prep,
                                    daug_dict=None,
                                    index_array=None,
                                    num_threads=1)    
    test_df.start()
    for i in range(training_iters):
         train_batch = df.next()
         sess.run([loss, acc], train_batch)
         if i == TEST_ITER:
             test_batch = test_df.next()
             sess.run([loss, acc], test_batch)
...
`
Does this methods make test data set and training data set share the same image preprocessing parameter such as mean of data set?

**Comment by lfwin** [comment]
I debugged two data_flow shared the same mean and variance..
many thanks for such good code!
